protocol/0056-REWA-rewards_overview.md

```diff
@@ -130,6 +130,32 @@ If a party **is not** a consensus or standby validator, their reward metric is s
 
 $$m_v = 0$$
 
+### Realised Returns metric
+
+The realised returns metric, , $m_{rz}$, measures the returns a party has realised across a number of epochs.
+
+Let $rz_i$ be a parties realised returns in an epoch. At the start of each epoch, the network will set $rz_i$ to `0`.
+
+During the epoch, each parties realised returns will be incremented as follows:
+
+- a party pays or receives a funding payment
+
+$$rz_i = rz_i + fundingPayment$$
```

Comment:
is this only funding payment or m2m wins/losses?

Reply:
Only funding payments.

Funding payments are considered "realised" M2M wins/losses are "unrealised" un till the position is closed or reduced at which point we do the logic described a few lines above.